controllers/admin/AdminStockMvtController.php

```diff
@@ -174,6 +174,9 @@ public function renderList()
             self::$currentIndex .= '&id_warehouse='.$id_warehouse;
         }
 
+        $this->_orderBy = 'a.date_add';
+        $this->_orderWay = 'DESC';
```

Comment:
Because it was so bad.if you want see last move, you have to go at the bottom of the page.., now it's first